docs/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "build.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "docs": "node docsServer.js",
```

Comment:
I made this change because when I ran `npm run docs`, the docs site was using an outdated version of ReactSearch.